keylime/tenant.py

```diff
@@ -302,8 +307,18 @@ def init_add(self, args):
                 self.supported_version = "1.0"
             else:
                 # Try to connect to the agent to get supported version
-                with RequestsClient(f"{self.agent_ip}:{self.agent_port}", tls_enabled=True, cert=self.agent_cert,
-                                    ignore_hostname=True, verify_custom=self.registrar_data['mtls_cert']) as get_version:
+                if self.registrar_data['mtls_cert']:
+                    self.verify_custom = self.registrar_data['mtls_cert']
+                    if self.registrar_data['mtls_cert'] == "disabled" :
+                        self.tls_agent_enabled = False
+                        self.verify_custom = False
+                        logger.warning(
+                            "Warning: mTLS for agents is disabled: the identity of each node will be based on the properties of the TPM only. "
```

Comment:
2 comments on this: can we make the "Warning" part strong by using all caps? And could we have separate error messages depending on whether `payload_script` is set or not?